src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -50,7 +50,7 @@
      data-toggle="tooltip"
      data-placement="top"
      style="display: inline-block; padding-right: 5px;">
-     <div class="btn-group margin-bottom-7px">
+    <div class="btn-group margin-bottom-7px">
```

Comment:
EDIT: I'm okay with leaving it as it's actually correcting the indentation. The wrong indentation was introduced recently in #7312.